Source/Core/Core/Core.cpp

```diff
@@ -62,6 +62,16 @@
 #include "VideoCommon/OnScreenDisplay.h"
 #include "VideoCommon/VideoBackendBase.h"
 
+// This can be removed when we move to VS2015
```

Comment:
Yeah. Originally the code didn't account for Android, so it made sense then. You're right though, I'm updating it.